src/modules/discovery.json

```diff
@@ -1624,6 +1624,18 @@
           "limit": {
             "type": "integer",
             "minimum": -1
+          },
+          "offeringType": {
```

Comment:
This makes sense to me, but I think we should pull `offeringType` into a shared definition, as it's used by WaysToWatch as well.

WaysToWatch lives here:
https://github.com/rdkcentral/firebolt-openrpc/blob/main/src/schemas/entertainment.json

and I suspect the new `OfferingType` definition would live in there as well.